pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -45,7 +48,7 @@ stdenv.mkDerivation rec {
     description = "Basic Linear Algebra Subprograms";
     license = licenses.bsd3;
     homepage = "https://github.com/xianyi/OpenBLAS";
-    platforms = with platforms; linux;
+    platforms = with platforms; all;
```

Comment:
Probably better to go with `unix` instead of `all`, since I doubt this fixes the package on cygwin, etc.